template.env

```diff
@@ -78,6 +76,14 @@ NB_API_QUERY_URL=http://XX.XX.XX.XX
 # NB_QUERY_TAG=latest
 # --------------------------------------
 
+# ---- SECURITY CONFIGURATION ----
```

Comment:
This is awesome, but once we merge the template it'll be the new default template for everyone, correct? 
So maybe we should add a comment here that the Authentication options are currently under active development and can be ignored unless you are developing or deploying a development version.